common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -275,8 +277,11 @@ describe 'Problem', ->
               callable()
             done: (callable) ->
               callable()
-        spyOn @problem.submitButtonLabel, 'text'
+        spyOn(@problem.submitButtonLabel, 'text')
+        $('#input_example_1').val('test').trigger('input')
```

Comment:
Sounds good, happy to keep the logic separate, will remove the unneeded calls for disabled/enabled.